addons/im_livechat/static/src/embed/common/chat_bubble_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.ChatBubble" t-inherit-mode="extension">
+        <xpath expr="//ImStatus" position="attributes">
+            <attribute name="t-if" add="!(thread.correspondent.im_status == 'im_partner' and env.embedLivechat)" separator=" and "/>
```

Comment:
I'd rather having a JS getter instead. It's easier to manipulate (imagine the previous condition is `A OR B`, you'd want it to be `(A OR B) AND C` but here, you'll got `A OR (B AND C)` which is not good nor easy to understand/maintain).